pkg/apis/sealed-secrets/v1alpha1/types.go

```diff
@@ -36,6 +37,8 @@ type SealedSecret struct {
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
 	Spec SealedSecretSpec `json:"spec"`
+
```

Comment:
Add a `// +optional` comment telling the autogen machinery that this is optional, just like: https://github.com/kubernetes/api/blob/99b35971cc9d07896d4428dc09ad3efacef8b21f/core/v1/types.go#L4613